hack/make-rules/build.sh

```diff
@@ -22,5 +22,7 @@ set -o pipefail
 
 KUBEEDGE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
 source "${KUBEEDGE_ROOT}/hack/lib/init.sh"
+source "${KUBEEDGE_ROOT}/hack/verify-golang.sh"
 
+kubeedge::golang::verify_golang_version
```

Comment:
this will check golang version twice.
we can delete line 44 `kubeedge::golang::verify_golang_version` in `hack/verify-golang.sh`

Reply:
```
# make all
hack/verify-golang.sh
go detail version: go version go1.17.6 linux/amd64
go version: 1.17.6
KUBEEDGE_OUTPUT_SUBPATH=_output/local hack/make-rules/build.sh 
go detail version: go version go1.17.6 linux/amd64
go version: 1.17.6
go detail version: go version go1.17.6 linux/amd64
go version: 1.17.6
building github.com/kubeedge/kubeedge/cloud/cmd/cloudcore
+ go build -o /root/go/src/github.com/kubeedge/kubeedge/_output/local/bin/cloudcore -gcflags= -ldflags '-s -w -buildid= -X github.com/kubeedge/kubeedge/pkg/version.buildDate=2022-01-13T11:48:53Z -X github.com/kubeedge/kubeedge/pkg/version.gitCommit=3b624bb5d71ea4430db432adfc0a19898397245e -X github.com/kubeedge/kubeedge/pkg/version.gitTreeState=clean -X github.com/kubeedge/kubeedge/pkg/version.gitVersion=v1.9.0-134+3b624bb5d71ea4 -X github.com/kubeedge/kubeedge/pkg/version.gitMajor=1 -X github.com/kubeedge/kubeedge/pkg/version.gitMinor=9+' github.com/kubeedge/kubeedge/cloud/cmd/cloudcore
```

Reply:
Done. Remove the kubeedge::golang::verify_golang_version in all script and remain in hack/verify-golang.sh. So `hack/verify-golang.sh` can also run.

Reply:
lgtm